### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by SimbaWANGU**
The following were achieved in this pull request:

- Personalize the content of your page. Choose a topic that is different than the one in the original design.
- Build only these 2 pages:
1. The home page.
2. The about page.
Each of these pages should have versions for 2 different screen sizes:
Mobile: up to 768px wide.
Desktop: 768px or wider.
Interactions

- Links
The home page has a link in the menu to the about page.
The logo in the header links to the home page.

- Mobile menu
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
The mobile menu should have a close (X) button that closes the menu.

- Dynamic page
The section "Featured speakers" should be created dynamically in JavaScript.
You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

**Comment by SimbaWANGU** [comment]
Hello Reviewer
I have submitted the wrong link
Kindly review the code in this pull request instead [https://github.com/S-imba/Kenyan-Creative---Microverse-Capstone-Project/pull/2](https://github.com/S-imba/Kenyan-Creative---Microverse-Capstone-Project/pull/2)

I have contacted student success to help me with this issue

File: public/about.css [hunk 1]
```diff
@@ -0,0 +1,633 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700;300;400;500&display=swap');
+
+* {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  list-style: none;
+}
```
Comment (uche-inyama) [inline public/about.css:7]: Kindly use "Cocogoose, Lato" fonts, in your font-family.

File: public/about.css [hunk 2]
```diff
@@ -0,0 +1,633 @@
...
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+}
+
+@media (max-width: 782px) {
+  body > hr {
+    display: none;
+  }
+
+  .header {
```
Comment (uche-inyama) [inline public/about.css:25]: I noticed you used both max-width & min-width to define your media query.
A mobile first approach is required in this project. Kindly remove the max-width definition.

If there is a need clarification, kindly reachout.

### :slightly_smiling_face:

File: public/index.css [hunk 3]
```diff
@@ -0,0 +1,658 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700;300;400;500&display=swap');
+
+* {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  list-style: none;
+}
+
```
Comment (uche-inyama) [inline public/index.css:8]: Kindly use "Cocogoose, Lato" fonts, in your font-family definition.

File: public/index.css [hunk 4]
```diff
@@ -0,0 +1,658 @@
...
+.header {
+  background-image: url('../assets/images/header_background.png');
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+}
+
+@media (max-width: 782px) {
+  .header {
+    background-color: rgba(220, 220, 220, 0.3);
+  }
```
Comment (uche-inyama) [inline public/index.css:23]: I noticed you used both max-width & min-width to define your media query.
A mobile first approach is required in this project. Kindly remove the max-width definition.

If there is a need clarification, kindly reachout.

### :slightly_smiling_face:

**Review by uche-inyama — COMMENTED** [review]
### Hi @S-imba :wave:  

Good job, your project is coming along just fine.

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes

- Going through your css files, I noticed you did not use any of these colors: "#272A31, #EC5242, #D3D3D3" as required in the design template.
- The contents under each header in your readme file needs to be updated, especially:
      - There is no link for a demo.
      - Kindly reference the source of your of design, under acknowledgement header. 
 - In the **featured speaker** section, the **margin** between the image and the write-up should be reduced.

#### For other mentions, kindly check the comments under the review. :slightly_smiling_face: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by SimbaWANGU** [comment]
I have completed the requested changes :)

**Pull request closed without merging**